src/serverExplorer.ts

```diff
@@ -346,7 +346,9 @@ export class ServerExplorer implements TreeDataProvider< Protocol.ServerState |
             const state: Protocol.ServerState = item as Protocol.ServerState;
             const handle: Protocol.ServerHandle = state.server;
             const id1: string = handle.id;
-            const runState: string = this.runStateEnum.get(state.state);
+            const runState: string = (state.state === 2 && state.runMode === 'debug') ?
```

Comment:
wouldn't it be more consistent to call this variable ```runMode```?

Reply:
well, not sure. runState can also have as a value "stopped" so if we change the name to runMode is like we're telling ... in which mode the server is running? and it doesn't make sense if the server is stopped. I think it's better runState or serverState. What do you think??